### PR TITLE
Fix tubular mesh

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Fix tubular mesh construction which could appear when a tube was oriented in an axis direction (thanks @naubry for the sequence bug). 
The face mesh generation of tubular mesh was also improved  by using distance measure instead direction.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [n.a] Doxygen documentation of the code completed (classes, methods, types, members...)
- [n.a] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -69,6 +69,12 @@
     was wrong on 3d and on closed Khalimsky space.
     (Bertrand Kerautret, [#1156](https://github.com/DGtal-team/DGtal/pull/1156))
 
+- *Shape Package*
+  - Fix a tubular mesh construction problem (missing faces) which appears
+    when the center line is oriented in an main axis direction (in
```
Comment (dcoeurjo) [inline ChangeLog.md]: an->a

File: ChangeLog.md [hunk 2]
```diff
@@ -69,6 +69,12 @@
...
+    createTubularMesh()). Also improve the face construction.
```
Comment (dcoeurjo) [inline ChangeLog.md]: improves and fixes ?

Comment (kerautret) [inline ChangeLog.md]: yes thanks done.

**Comment by dcoeurjo** [comment]
Ok thanks. Merging

**Pull request merged**